reliontomo/__init__.py

```diff
@@ -32,19 +34,23 @@
     import relion
 
     class Plugin(relion.Plugin):
-        _supportedVersions = [V4_0]
+        _supportedVersions = [V5_0]
```

Comment:
Aren't we going to support relion 4?

Reply:
We're, at least during some time. Let me fix it. Thanks, @pconesa